template/en/default/admin/params/admin.html.tmpl

```diff
@@ -23,6 +23,14 @@
    desc = "Set up account policies"
 %]
 
+[% rate_limit_rules_desc = BLOCK %]
+This parameter is a json object that allows for overriding the default rate limit.
+Each key takes an array in the form of [MAX_RATE, SECONDS] where MAX_RATE is the maximum
+number of requests that can occur over SECONDS. The default is [75, 60] or 75 requests
+over 60 seconds. Valid keys are <code>get_b[%''%]ug</code> which covers JSONRPC, XMLRPC, REST and BZAPI single
```

Comment:
As an outside, I'd love to understand more about what these ```[%''%]``` blocks are defending against.